src/base/init/install.c

```diff
@@ -237,12 +253,23 @@ static void call_installdos(void)
 		free(boot_dir_path);
 		return;
 	}
-	/* symlink command.com in case someone hits Shift or F5 */
-	ret = asprintf(&system_str,
-			"ln -s ../drives/d/command.com "
-			"../drives/d/kernel.sys "
-			"\"%s\"",
-			boot_dir_path);
+	switch (dos_status) {
+		case 0:
+			/* symlink command.com in case someone hits Shift or F5 */
+			ret = asprintf(&system_str,
+					"ln -s ../drives/d/command.com "
+					"../drives/d/kernel.sys "
+					"\"%s\"",
+					boot_dir_path);
+			break;
+		case 100:
+			ret = asprintf(&system_str,
+					"ln -s ../drives/d/COMMAND.COM "
```

Comment:
Yeah, that's indeed annoying. The reason is now due to mtools (mcopy) which seems to use uppercase. I couldn't find a setting to switch it to lowercase. I'm aware that dosemu uses lowercase. It would be nice if mtools and dosemu could use the same casing. Otherwise I'd have to look for a command to rename files en masse to lowercase.

I used 7zip before to extract disk images, but mcopy manages to handle some disks that 7zip can't.